test/e2e/e2e_test.go

```diff
@@ -177,51 +179,54 @@ func runPodLifetimeStrategy(
 		t.Fatalf("%v", err)
 	}
 
-	maxPodLifeTimeSeconds := uint(1)
-	strategy := deschedulerapi.DeschedulerStrategy{
-		Enabled: true,
-		Params: &deschedulerapi.StrategyParameters{
-			PodLifeTime:                &deschedulerapi.PodLifeTime{MaxPodLifeTimeSeconds: &maxPodLifeTimeSeconds},
-			Namespaces:                 namespaces,
-			ThresholdPriority:          priority,
-			ThresholdPriorityClassName: priorityClass,
-			LabelSelector:              labelSelector,
-		},
-	}
+	podEvictor := evictions.NewPodEvictor(
+		clientset,
+		evictionPolicyGroupVersion,
+		false,
+		nil,
+		maxPodsToEvictPerNamespace,
+		nodes,
+		false,
+		&events.FakeRecorder{},
+	)
 
-	thresholdPriority, err := utils.GetPriorityFromStrategyParams(ctx, clientset, strategy.Params)
-	if err != nil {
-		t.Fatalf("Failed to get threshold priority from strategy's params")
+	var thresholdPriority int32
+	if priority != nil {
+		thresholdPriority = *priority
+	} else {
+		thresholdPriority, err = utils.GetPriorityFromPriorityClass(ctx, clientset, priorityClass)
+		if err != nil {
+			t.Fatalf("Failed to get threshold priority from plugin arg params")
+		}
 	}
 
-	eventRecorder := &events.FakeRecorder{}
-
-	strategies.PodLifeTime(
-		ctx,
-		clientset,
-		strategy,
+	evictorFilter := evictions.NewEvictorFilter(
 		nodes,
-		evictions.NewPodEvictor(
-			clientset,
-			evictionPolicyGroupVersion,
-			false,
-			nil,
-			maxPodsToEvictPerNamespace,
-			nodes,
-			false,
-			eventRecorder,
-		),
-		evictions.NewEvictorFilter(
-			nodes,
-			getPodsAssignedToNode,
-			false,
-			evictCritical,
-			false,
-			false,
-			evictions.WithPriorityThreshold(thresholdPriority),
-		),
 		getPodsAssignedToNode,
+		true,
```

Comment:
Has anything changed that requires `evictLocalStoragePods=true`?

Reply:
No, just copy-pasta ©️